requirements/base.txt

```diff
@@ -180,7 +178,6 @@ stevedore==5.4.1
 typing-extensions==4.12.2
     # via
     #   edx-opaque-keys
-    #   elasticsearch-dsl
```

Comment:
Author's note: I'm not sure why this transitive dependency comment was removed when I ran `make compile-requirements` locally, but it shouldn't have any effect. (Might be a Mac vs. Linux difference, or a leftover of someone manually editing the files at some point.)